### PR TITLE
External search prototype

### PR DESCRIPTION
**Pull request by abulte**
Designed to work with https://github.com/abulte/ecospheres-search-index, based on Meilisearch.

Not sure if we'll go with this but I guess it shows it can be quite simple, at least for basic features.

### CODE REVIEW

**Review by streino — COMMENTED** [review]
Meilisearch looks cool!
Except for complex geo search (bbox, polygon), looks like it has everything we'd need.

File: index.html [hunk 1]
```diff
@@ -4,7 +4,7 @@
     <meta charset="UTF-8">
     <link rel="icon" href="/favicon.ico">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Vite App</title>
+    <title>Ecosphères</title>
```
Comment (streino) [inline index.html:7]: Now it looks real ;)

File: src/views/datasets/DatasetsListView.vue [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+        type="dataset"
+        :alt-img="d.title"
+        :link="`/datasets/${d.slug}`"
+        :title="d.title"
+        :description="d.description"
+        :img="d.organization.logo"
+      />
+    </div>
+  </div>
+  <DsfrPagination v-if="pages.length" :current-page="currentPage - 1" :pages="pages" @update:current-page="onUpdatePage" />
```
Comment (streino) [inline src/views/datasets/DatasetsListView.vue:39]: This doesn't do anything at this point, right?

**Review by streino — APPROVED** [review]

File: src/views/datasets/DatasetsListView.vue (hunk 2 above)
Comment (abulte) [reply src/views/datasets/DatasetsListView.vue:39]: Right. I'll remove it since all variables are not initialized.

**Pull request merged**